lib/services/configuration-files.js

```diff
@@ -0,0 +1,113 @@
+import { VERSIONED_CONFIGURATION_WATCHER, VERSIONED_CONFIGURATION_ROOT } from '../../env';
+import moment from 'moment';
+import { SemanticFile } from '../entities/semantic-file';
+import { DirectorySynchronizer } from './directory-synchronizer';
+import { Gaze } from 'gaze';
+import { sleep } from '../util/generic';
+import { SemanticFormSpecification } from '../entities/semantic-form-specification';
+
+/**
+ * Service providing the up-to-date and synced-up configuration-files.
+ */
+export class ConfigurationFiles {
+
+  constructor() {
+    this.directory = new DirectorySynchronizer(VERSIONED_CONFIGURATION_ROOT);
+    this.debug = true;
```

Comment:
Why not use `DEBUG_LOGS` here ?

Reply:
In all honest think I wrongly used the name "debug" here as it is more referring to logging that should be always there.